src/Timer.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+#include <chrono>
+
+using minutes = std::chrono::seconds;  // To simulate time flow ;)
```

Comment:
You have this line in both Pizza.hpp and Timer.hpp. Is it neccesary?

Reply:
![image](https://user-images.githubusercontent.com/57813618/93077403-0a2e9980-f689-11ea-980d-9eaa0e1607d4.png)
Interesting stuff, I cleaned this file and pushed it into github without that stuff :P